src/systemctl/systemctl.c

```diff
@@ -85,6 +85,23 @@
 #include "verbs.h"
 #include "virt.h"
 
+/* The init script exit status codes
+   0       program is running or service is OK
+   1       program is dead and /var/run pid file exists
+   2       program is dead and /var/lock lock file exists
+   3       program is not running
+   4       program or service status is unknown
+   5-99    reserved for future LSB use
+   100-149 reserved for distribution use
+   150-199 reserved for application use
+   200-254 reserved
+*/
+#define PROGRAM_RUNNING_OR_SERVICE_OK         0
+#define PROGRAM_DEAD_AND_PID_EXISTS           1
+#define PROGRAM_DEAD_AND_LOCK_FILE_EXISTS     2
+#define PROGRAM_NOT_RUNNING                   3
+#define PROGRAM_OR_SERVICES_STATUS_UNKNOWN    4
```

Comment:
And maybe a nice comment here would be nice, so that Lennart isn't confused the nex ttime he looks at it this:

```c
/* Note that these are the *status* exit codes as defined in the LSB spec. The *start* exit codes are defined by LSB are different and already have definitions in exit-code.h */
```

or so....